### ISSUE TITLE
Submitting comments returns "Network error"

### ISSUE THREAD

**Issue by scrutinizer11** [ExistOrLive/GithubClient#79]
It's impossible to submit comments. The error "Network error" is popping up, likewise sending feedback about the app in the app's _Feedback_ section.

P.S. I'm posting this about **ZLGithubClient**. Judging from this repo's name I'm confusing whether it's the same app.

**Comment by ExistOrLive** [comment]
Yes，this is repo for “ZLGithubClient”; The error "Network error” should be some problem about network. Check your wifi or mobile network please.

**Comment by scrutinizer11** [comment]
No, connectivity has nothing to do with it. It's active. Other apps have no problems with the Internet.
